### DISCUSSION TITLE
Let's discuss about the GPIOViewer Library and its Web Application

### DISCUSSION THREAD

**Discussion by thelastoutpostworkshop** [thelastoutpostworkshop/gpio_viewer#86] [Announcements]
## 👋 Welcome to the GPIOViewer Discussion!
  We’re using Discussions as a place to connect with other users of the GPIOViewer. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other users.
  * Welcome others and be open-minded.

**Comment by Teketa** [comment]
I would love to see it be expanded to other boards, such as the Raspberry Pi Pico(W)!

**Comment by toetoast** [comment]
I'm really loving the new UI design with version 1.5, especially the new drop-down menu full of cool features - which was a pleasant surprise!

Keep up the great work

Regards
Tony

**Reply by thelastoutpostworkshop** [reply]
Thank you

**Comment by toetoast** [comment]
I do a lot of 3D printing using the Prusa Connect Web Application, and I often get the following screen when a new Web App version is available.  I was wondering if there would be some value in doing this for the GPIOViewer Web Application, so that anyone with the screen open will get a message asking if they want to update to the latest version or not?

Regards
Tony
![image (1)](https://github.com/thelastoutpostworkshop/gpio_viewer/assets/65843197/6114c403-ecc2-4a81-9de6-44cf8943ff37)

**Comment by thelastoutpostworkshop** [comment]
Very good idea!
